src/services/repos/ReposTeamQueryOpts.test.tsx

```diff
@@ -178,13 +197,10 @@ describe('useReposTeam', () => {
             {
               repos: [repo3, repo4],
               isCurrentUserPartOfOrg: true,
-              pageInfo: {
-                hasNextPage: false,
-                endCursor: 'aa',
-              },
+              pageInfo: { hasNextPage: false, endCursor: 'aa' },
             },
           ],
-          pageParams: [undefined, 'MjAyMC0wOC0xMSAxNzozMDowMiswMDowMHwxMDA='],
```

Comment:
This behavior of returning `''` instead of `undefined` seems new and tricky - is that just this test behaving differently now, or has pageParams changed?

Reply:
In TSQ V5 you're required to provide a value to `initialPageParams` which cannot be undefined so we pass an empty string instead

Reply:
Docs: https://tanstack.com/query/latest/docs/framework/react/reference/useInfiniteQuery

Reply:
nice, thanks!